pkg/detectors/appfollow/appfollow.go

```diff
@@ -22,7 +20,7 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"appfollow"}) + `\b([0-9A-Za-z]{20})\b`)
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"appfollow"}) + `\b([\w-]*\.[\w-]*\.[\w-]*)\b`)
```

Comment:
@lc since this is a JWT I think we can tighten up the regex a bit. See https://github.com/trufflesecurity/trufflehog/blob/main/pkg/detectors/formio/formio.go#L23 for an example

Reply:
I can't speak to AppFollow's format, but based on my testing you can really only set a minimum for each fragment. The payload and signature can be quite long (>1,000 chars), and [as mentioned elsewhere](https://github.com/gitleaks/gitleaks/pull/1214#issuecomment-1685136432) the regexp package only allows you to specify <=1,000 chars.

Reply:
tightened it up @zricethezav:

![CleanShot 2023-10-23 at 12 24 03](https://github.com/trufflesecurity/trufflehog/assets/19563282/c9b45ac2-feed-4d34-9249-f3cc17c6160a)